scripts/ci/check-version-contract.py

```
@@ -39,17 +40,33 @@ def read_version_contract(repo_root: Path) -> dict[str, str]:
     }
 
 
+# Что вообще считается версией выпуска.
+#
+# Предвыпуск законен: замерить доставку можно только на настоящем релизе, а
```

Ruff (0.16.1)

[warning] Comment contains ambiguous `а` (CYRILLIC SMALL LETTER A). Did you mean `a` (LATIN SMALL LETTER A)?

(RUF003)